SpatialGDK/Source/SpatialGDK/Private/LoadBalancing/WorkerRegion.cpp

```diff
@@ -23,16 +26,56 @@ AWorkerRegion::AWorkerRegion(const FObjectInitializer& ObjectInitializer)
 	SetRootComponent(Mesh);
 }
 
-void AWorkerRegion::Init(UMaterial* Material, const FColor& Color, const float Opacity, const FBox2D& Extents, const float Height,
-						 const float VerticalScale)
+void AWorkerRegion::Init(UMaterial* BackgroundMaterial, UMaterial* InCombinedMaterial, UFont* InWorkerInfoFont, const FColor& Color,
+						 const float Opacity, const FBox2D& Extents, const float Height, const float VerticalScale,
+						 const FString& InWorkerInfo, const bool bInEditor)
 {
+	// Background translucent coloured worker material
+	BackgroundMaterialInstance = UMaterialInstanceDynamic::Create(BackgroundMaterial, nullptr);
 	SetHeight(Height);
 
-	MaterialInstance = UMaterialInstanceDynamic::Create(Material, nullptr);
-	Mesh->SetMaterial(0, MaterialInstance);
+	if (bInEditor)
+	{
+		// In editor, display the basic boundary material
+		Mesh->SetMaterial(0, BackgroundMaterialInstance);
```

Comment:
Does this mean we're only seeing the text at runtime? Makes sense, although could be that seeing the virtual worker IDs would be useful, would be def out of scope for this PR anyway 👍

Reply:
This approach to dynamically render the text on the material only works at runtime as the UCanvasRenderTarget2D->OnCanvasRenderTargetUpdate() callback is only triggered at runtime.